Chat/Infrastructure/Repositories/MessageRepository.cs

```diff
@@ -0,0 +1,77 @@
+﻿using Domain.Entities;
+using Domain.Interfaces;
+using Infrastructure.Data;
+using Microsoft.EntityFrameworkCore;
+
+
+namespace Infrastructure.Repositories
+{
+    public class MessageRepository : IMessageRepository
+    {
+
+        private readonly DatabaseContext _db;
+
+        public MessageRepository(DatabaseContext db)
+        {
+            _db = db;
+        }
+
+        public async Task ChangeMessageStatusAsync(int chatid, int id, string status)
```

Comment:
Second word in the parameter name should start with a capital letter. Check and correct this throughout your application